example/seed-crds/10-crd-extensions.gardener.cloud_dnsrecords.yaml

```diff
@@ -131,6 +131,11 @@ spec:
                         description: ErrorCode is a string alias.
                         type: string
                       type: array
+                    lastHeartbeatTime:
```

Comment:
All these changes here must also be updated in https://github.com/gardener/gardener/tree/master/pkg/operation/botanist/component/extensions/crds/templates